guides/common/modules/proc_recovering-from-disaster-with-two-active-project-servers.adoc

```diff
@@ -0,0 +1,22 @@
+[id="recovering-from-disaster-with-two-active-project-servers"]
+= Recovering from disaster with two active {ProjectServer}s
+
+If the health checks implemented in xref:preparing-for-disaster-recovery-with-two-active-project-servers[] report an issue on one of your {ProjectServer}s, it might mean that the server has failed.
+If the server is down, you must re-register hosts to the other server.
```

Comment:
Same comment as above. No need for "re-register" when "register" works in this context.